Error in sys.excepthook:
Traceback (most recent call last):
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/linecache.py", line 46, in getlines
    return updatecache(filename, module_globals)
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/linecache.py", line 136, in updatecache
    with tokenize.open(fullname) as fp:
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/tokenize.py", line 392, in open
    buffer = _builtin_open(filename, 'rb')
KeyboardInterrupt
Original exception was:
Traceback (most recent call last):
  File "/Users/sigurdrollsolberg/Documents/School/MasterPublication/DDPN/modenet.py", line 126, in <module>
    training_loop.train(train_loader=train_loader, val_loader=val_loader, epochs=200, verbose=True)
  File "/Users/sigurdrollsolberg/Documents/School/MasterPublication/DDPN/src/training_tools.py", line 114, in train
    train_loss, train_accuracy = self.test(train_loader, sample = True)
  File "/Users/sigurdrollsolberg/Documents/School/MasterPublication/DDPN/src/training_tools.py", line 145, in test
    output = self.model(data)
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/sigurdrollsolberg/Documents/School/MasterPublication/DDPN/src/models.py", line 31, in forward
    x = torch.sum(self.inner_transform(x), dim = -2)
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/site-packages/torch/nn/modules/container.py", line 204, in forward
    input = module(input)
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/sigurdrollsolberg/Documents/School/MasterPublication/DDPN/modenet.py", line 74, in forward
    x = self.Gamma(x)
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/sigurdrollsolberg/opt/anaconda3/envs/aml_2/lib/python3.9/site-packages/torch/nn/modules/linear.py", line 114, in forward
    return F.linear(input, self.weight, self.bias)
KeyboardInterrupt